backend/apps/gateway/src/app.module.ts

```diff
@@ -18,27 +38,14 @@ import { Service } from '@app/interservice-api/services';
     AuthController,
     UserController,
     LanguagesController,
+    CollaborationController,
   ],
   providers: [
     GoogleOauthStrategy,
-    {
-      provide: Service.QUESTION_SERVICE,
-      useFactory: (configService: ConfigService) => {
-        const questionServiceOptions = configService.get(
-          'questionServiceOptions',
-        );
-        return ClientProxyFactory.create(questionServiceOptions);
-      },
-      inject: [ConfigService],
-    },
-    {
-      provide: Service.USER_SERVICE,
-      useFactory: (configService: ConfigService) => {
-        const userServiceOptions = configService.get('userServiceOptions');
-        return ClientProxyFactory.create(userServiceOptions);
-      },
-      inject: [ConfigService],
-    },
+    YjsGateway,
```

Comment:
Let's explore shifting the websocket connection to a separate gateway in the future